app/src/main/java/com/github/studydistractor/sdp/account/UserInterface.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.studydistractor.sdp.account
+
+import androidx.compose.runtime.snapshots.SnapshotStateList
+
+interface UserInterface {
```

Comment:
I agree with Yohan on his comments in my PR on how the name things. And this interface should just be called `User` for me.

Reply:
This will cause issues with Adrien he made a class incompatible also called User. I think it is the best compromise.

Reply:
Maybe we should call it Friends or something related then no ?